slick/src/sphinx/developers.rst

```diff
@@ -0,0 +1,155 @@
+For Slick developers
+======================
+
+`slick at github <https://github.com/slick/slick>`_
+
+`waffle board <https://waffle.io/slick/slick>`_
+
+`slick-extensions at github <https://github.com/typesafe-hub/slick-extensions>`_
```

Comment:
This URL looks to be dead - and I don't think we need to talk about extensions anymore, so probably just delete this line.